pom.xml

```diff
@@ -561,6 +561,7 @@
         <com.h2database.version>1.4.196</com.h2database.version>
         <!-- DAO dependencies -->
         <javax.persistence.version>2.2.0</javax.persistence.version>
```

Comment:
shall we get rid of javax.persistence.version property?

Reply:
+1

Reply:
The latest version of [javax.persistence](http://repo.maven.apache.org/maven2/org/eclipse/persistence/javax.persistence/) is 2.2.0 and [for org.eclipse.persistence.core](http://repo.maven.apache.org/maven2/org/eclipse/persistence/org.eclipse.persistence.core/) is 2.7.1
Hence we need to have both.